assets/js/romo/base.js

```diff
@@ -42,18 +42,25 @@ Romo.prototype.parent = function(childElem) {
   return childElem.parentNode;
 }
 
-Romo.prototype.siblings = function(elem) {
-  return Array.prototype.filter.call(elem.parentNode.children, function(childElem) {
-    return childElem !== elem;
-  });
-}
-
-Romo.prototype.prev = function(fromElem) {
-  return fromElem.previousElementSibling;
-}
-
-Romo.prototype.next = function(fromElem) {
-  return fromElem.nextElementSibling;
+Romo.prototype.parents = function(childElem, selector) {
+  var parentElem = this.parent(childElem);
+  if (parentElem) {
+    if (!selector || Romo.is(parentElem, selector)) {
+      if (Romo.is(parentElem, 'body')) {
+        return [parentElem];
+      } else {
+        return [parentElem].concat(this.parents(parentElem, selector));
+      }
+    } else {
+      if (Romo.is(parentElem, 'body')) {
+        return [];
```

Comment:
@kellyredding - It seems inconsistent to not return the `<body>` elem here but with the selector it can? I don't know of any issues related to this but it seems more consistent to me that the `parents` method either does or doesn't return the `<body>` elem regardless of a selector being passed.

Reply:
@kellyredding - Thanks for explaining, I didn't interpret this correctly. I'm good with this.